app/controllers/articles_controller.rb

```diff
@@ -24,4 +24,36 @@ def update
     #WIP
   end
 
+  def publish
+    article = prepare_article(params[:id])
```

Comment:
`prepare_article` in this place loads more data each time and reloads it after the update, what is the reason for that?